frontend/lib/parsing/chpl.ypp

```diff
@@ -929,6 +913,10 @@ opt_attribute_actuals:
   {
     $$ = $2;
   }
+  |
```

Comment:
This looks like it allows `@attribute` without parentheses.  Was that intentional?  Or am I misinterpreting the change here?

Reply:
it allows attributes without parentheses, but not with arguments. So you can do a paren-less attribute so long as there are no arguments supplied, e.g. `@chpldoc.nodoc`